src/main/java/com/exactpro/epfast/template/UnicodeStringField.java

```diff
@@ -0,0 +1,13 @@
+package com.exactpro.epfast.template;
+
+public interface UnicodeStringField extends FieldInstruction {
+
+    Presence getPresence();
+
+    FieldOperator getOperator();
+
+    Charset getCharset();
```

Comment:
Charset isn't needed in string interfaces: unicode string always is unicode and ascii - always ascii. Charset enum can be removed as well (actually it still be needed for xml reader but it should be there, not in interfaces)